src/drivers/imx/interrupt-noirq.c

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright 2021 NXP
+//
+// Author: Peng Zhang <peng.zhang_8@nxp.com>
+
+#include <sof/common.h>
+#include <sof/drivers/interrupt.h>
+#include <sof/lib/cpu.h>
+#include <sof/lib/io.h>
+#include <sof/lib/memory.h>
+#include <sof/lib/uuid.h>
+#include <sof/list.h>
+#include <sof/spinlock.h>
+#include <errno.h>
+#include <inttypes.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+/* fa00558c-d653-4851-a03a-b21f125a9524 */
+DECLARE_SOF_UUID("noirq-imx", noirq_imx_uuid, 0xfa00558c, 0xd653, 0x4851,
+		 0xa0, 0x3a, 0xb2, 0x1f, 0x12, 0x5a, 0x95, 0x24);
+
+DECLARE_TR_CTX(noirq_i_tr, SOF_UUID(noirq_imx_uuid), LOG_LEVEL_INFO);
+
+int irqstr_get_sof_int(int irqstr_int)
+{
+	return irqstr_int;
+}
+
+void platform_interrupt_init(void)
+{
+	return;
```

Comment:
I find an empty blocks clearer and faster to read. I think they're more common here.